ode-wrapper/src/test/java/com/trihydro/odewrapper/helpers/SetItisCodesTest.java

```diff
@@ -102,7 +99,7 @@ public void setItisCodesRc_translated() {
         WydotTimRc tim = new WydotTimRc();
         ArrayList<String> itisCodes = new ArrayList<>();
         itisCodes.add("4868");
-        itisCodes.add("769");
+        itisCodes.add("770");
```

Comment:
Question (blocking): Should we be expecting 769 or 770 here? I believe Rick has a preference for 769, so you may want to check with him here.

Reply:
Discussed offline, this ITIS code is self-contained to the unit test and doesn't indicate usage in the system; we may re-implement the mapping in the future to map 770 to 769 in which case it would make sense to use 770 in our unit tests.